### PR TITLE
W-13005270: Split SchedulerThreadPoolsTestCase in uber/dedicated

### PR DESCRIPTION
**Pull request by elrodro83**

### CODE REVIEW

File: src/test/java/org/mule/service/scheduler/internal/UberSchedulerThreadPoolsTestCase.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import org.mule.runtime.api.scheduler.Scheduler;
+
+import java.util.concurrent.ExecutionException;
+
+import org.junit.Test;
+
+import io.qameta.allure.Description;
+import io.qameta.allure.Issue;
+
+public class UberSchedulerThreadPoolsTestCase extends SchedulerThreadPoolsTestCase {
```
Comment (fsgonz) [inline src/test/java/org/mule/service/scheduler/internal/UberSchedulerThreadPoolsTestCase.java:21]: Add feature annotation.

File: src/test/java/org/mule/service/scheduler/internal/DedicatedSchedulerThreadPoolsTestCase.java [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeoutException;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.Consumer;
+
+import org.junit.Test;
+
+import io.qameta.allure.Description;
+
+public class DedicatedSchedulerThreadPoolsTestCase extends SchedulerThreadPoolsTestCase {
```
Comment (fsgonz) [inline src/test/java/org/mule/service/scheduler/internal/DedicatedSchedulerThreadPoolsTestCase.java:41]: Add Same allure Feature annotation as in the original test.

**Review by fsgonz — CHANGES_REQUESTED** [review]

**Review by fsgonz — APPROVED** [review]

File: src/test/java/org/mule/service/scheduler/internal/DedicatedSchedulerThreadPoolsTestCase.java (hunk 2 above)
Comment (elrodro83) [reply src/test/java/org/mule/service/scheduler/internal/DedicatedSchedulerThreadPoolsTestCase.java:41]: allure annotations are inherited

File: src/test/java/org/mule/service/scheduler/internal/UberSchedulerThreadPoolsTestCase.java (hunk 1 above)
Comment (elrodro83) [reply src/test/java/org/mule/service/scheduler/internal/UberSchedulerThreadPoolsTestCase.java:21]: allure annotations are inherited

**Pull request merged**